feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableRoom.kt

```diff
@@ -20,6 +21,8 @@ internal val TimetableRoom.color: Color
             RoomC -> colors.hallC
             RoomD -> colors.hallD
             RoomE -> colors.hallE
+            // The color of D is set as workaround.
+            RoomDE -> colors.hallD
```

Comment:
I would like to assign a room color for the third day or a gradient color for D and E if possible, but I have decided that it is not advisable to make major changes to this structure now.